dig.go

```diff
@@ -265,10 +256,50 @@ var _graphTmpl = template.Must(
 // Visualize parses the graph in Container c into DOT format and writes it to
 // io.Writer w.
 func Visualize(c *Container, w io.Writer, opts ...VisualizeOption) error {
-	if err := _graphTmpl.Execute(w, c.dg); err != nil {
-		return fmt.Errorf("error executing template: %v", err)
+	return _graphTmpl.Execute(w, c.createGraph())
+}
+
+// VisualizeError parses the container and errors included in err into a DOT
+// format graph and writes it to io.Writer w.
+func VisualizeError(err error, w io.Writer, opts ...VisualizeOption) error {
+	ep, ok := err.(errEntryPoint)
+	if !ok {
+		return fmt.Errorf("no graph included in error: %v", err)
 	}
-	return nil
+
+	var errors []errVisualizer
+
```

Comment:
nit: delete empty line. The contents of the declared  `errors` are being built
in the block so they should be together.